__tests__/PetConstructor.test.js

```diff
@@ -0,0 +1,225 @@
+const { Pet, petConfig } = require('../src/PetConstructor');
```

Comment:
As this is so similar to the `PetClass.test.js` file theres no need to write the same comments again here, should be refactored in the same way